server/user_account/serializers.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework import serializers
+from .models import CustomUser
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
+        fields = ['id', 'user_name', 'first_name', 'last_name', 'password', 'role', 'email']
+
+    # overriding save method
```

Comment:
Great, can you explain why you are overriding it though?

Reply:
I'm overriding it to add hashing to the password through the set_password() method. If I don't do that the elements will be saved as they are. So once I override this method I can manipulate whatever was sent in the request before storing them in the database.

Reply:
I mean in the comment for the method

Reply:
Oh I see what you mean, yes I can add that.